Source/Core/Core/PowerPC/Jit64/Jit_LoadStore.cpp

```diff
@@ -307,34 +311,16 @@ void Jit64::dcbz(UGeckoInstruction inst)
 	int a = inst.RA;
 	int b = inst.RB;
 
-	u32 mem_mask = Memory::ADDR_MASK_HW_ACCESS;
-
-	// The following masks the region used by the GC/Wii virtual memory lib
-	mem_mask |= Memory::ADDR_MASK_MEM1;
-
 	MOV(32, R(RSCRATCH), gpr.R(b));
 	if (a)
 		ADD(32, R(RSCRATCH), gpr.R(a));
 	AND(32, R(RSCRATCH), Imm32(~31));
-	TEST(32, R(RSCRATCH), Imm32(mem_mask));
-	FixupBranch slow = J_CC(CC_NZ, true);
 
-	// Should this code ever run? I can't find any games that use DCBZ on non-physical addresses, but
-	// supposedly there are, at least for some MMU titles. Let's be careful and support it to be sure.
-	SwitchToFarCode();
-	SetJumpTarget(slow);
 	MOV(32, M(&PC), Imm32(jit->js.compilerPC));
 	BitSet32 registersInUse = CallerSavedRegistersInUse();
 	ABI_PushRegistersAndAdjustStack(registersInUse, 0);
-	ABI_CallFunctionR((void *)&Memory::ClearCacheLine, RSCRATCH);
+	ABI_CallFunctionR((void *)&PowerPC::ClearCacheLine, RSCRATCH);
```

Comment:
Inlined BAT checks are possible, but it would require a bit more infrastructure to make them efficient (a mapping from logical addresses to host pointers, or something like that).  It's on my TODO list.